README.md

```diff
@@ -1,4 +1,7 @@
 # Tracker-server
+Pusty projekt został wygenerowany za pomocą https://start.spring.io/. Jest realizowany w strukturze Maven, testy jednostkowe wykonujemy za pomocą JUnit. Używamy jacoco, jest realizowany jaco stage mavena.
```

Comment:
"Jest realizowany w strukturze Maven" - to chyba nie jest poprawne sformułowanie, co miałoby ono oznaczać? Może warto dodać wersję JUnita jeśli już o nim wspominamy, wersję jococo ewentualnie też. Co oznacza "jest realizowany jaco stage mavena"?